authentication/urls.py

```diff
@@ -40,4 +41,9 @@
     path('reviewer/<int:reviewer_id>/',
          UserReviewsView.as_view(), name='user-reviews'),
     path('logout/', LogoutView.as_view(), name='logout'),
+    path('saved-card/',
```

Comment:
hey @Niyitangasam, since this endpoint returns a list, consider making this path `/saved-cards`

Reply:
Hey @CryceTruly, thanks worked on that.